SmartDeviceLink-iOS/SmartDeviceLink/SDLProtocol.m

```diff
@@ -28,6 +28,7 @@ @interface SDLProtocol () {
     dispatch_queue_t _sendQueue;
     SDLPrioritizedObjectCollection *_prioritizedCollection;
     NSMutableDictionary *_sessionIDs;
+    NSMutableDictionary *_hashIDs;
```

Comment:
Do you think that instead of storing multiple dictionaries with sessionIDs / hashIDs / whatever else in the future, it should just be one dict with various metadata about the service? That's what I'm starting to lean towards.